arb/load_file.c

```diff
@@ -0,0 +1,35 @@
+/*
+    Copyright (C) 2019 Julian Rüth
+
+    This file is part of Arb.
+
+    Arb is free software: you can redistribute it and/or modify it under
+    the terms of the GNU Lesser General Public License (LGPL) as published
+    by the Free Software Foundation; either version 2.1 of the License, or
+    (at your option) any later version.  See <http://www.gnu.org/licenses/>.
+*/
+
+#include <stdio.h>
+#include <string.h>
+
+#include "arb.h"
+
+int arb_load_file(arb_t x, FILE* stream)
+{
+    char* data = NULL;
+    size_t length = 0;
+    
+    ssize_t nread = getdelim(&data, &length, ' ', stream);
```

Comment:
This is a GNU extension generally available since POSIX 2008. Can I expect this to be around? If it's not, I would prefer just to disable this function completely. It's really annoying to implement this (read characters until you run out of space, then realloc,…)

Reply:
If I understand things correctly, this became generally available with glibc 2.10 released 2009-05-09.

Reply:
What about MSVC?

Reply:
Right. I hadn't checked and it appears that they don't have it.